Makefile

```diff
@@ -43,6 +43,13 @@ privacyidea:
 	#(cd ${BUILDDIR_PI}; git checkout v${GIT_VERSION})
 	#(cd ${BUILDDIR_PI}; git submodule init; git submodule update --recursive --remote)
 	(cd ${BUILDDIR_PI}; rm -fr tests)
+	# --- Build new WebUI (requires npm/node) ---
+	@command -v npm >/dev/null 2>&1 || { echo >&2 "ERROR: npm not found. Please install nodejs & npm for the build host."; exit 1; }
+	(cd ${BUILDDIR_PI}/privacyidea/static_new; npm ci)
+	(cd ${BUILDDIR_PI}/privacyidea/static_new; npm run-script ng build)
+	# Drop node_modules from source tree to keep package clean
+	(cd ${BUILDDIR_PI}/privacyidea/static_new; rm -rf node_modules)
```

Comment:
I would go ahead an remove everything except the `dist` folder from `static_new` since we do not need the sources in the final package.